plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -535,6 +536,40 @@ public void testAddColumnWithCommentSpecialCharacter(String comment)
         }
     }
 
+    @Test
+    public void testAddColumnWithDecimal()
+    {
+        testAddColumnWithDecimal(DecimalType.createDecimalType(14, 5));
+        testAddColumnWithDecimal(DecimalType.createDecimalType(35, 5));
```

Comment:
You can simply use String value like `decimal(14, 5)` instead of building `DecimalType`.

Reply:
done~